packages/teams-js/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './private/index';
 export * from './public/index';
+export { UUID } from './internal/uuidObject';
```

Comment:
Wouldn't exporting this `internal` function defeat the purpose of it being `internal`? Should we consider moving this function to public instead?